Xamarin.Forms.Platform.iOS/CollectionView/GroupableItemsViewController.cs

```diff
@@ -122,10 +120,7 @@ string DetermineViewReuseId(DataTemplate template)
 		{
 			if (template == null)
 			{
-				// No template, fall back the the default supplemental views
-				return ItemsViewLayout.ScrollDirection == UICollectionViewScrollDirection.Horizontal
-					? HorizontalDefaultSupplementalView.ReuseId
-					: VerticalDefaultSupplementalView.ReuseId;
+				return EmptySupplementalView.ReuseId;
```

Comment:
I see you removed the whole horizontal/vertical logic. How does this behave when in horizontal mode?

Reply:
Same applies for horizontal mode... Attaching screentshots, see the `Xamarin.Forms.Controls.Issues.CategoryGroup` at center of screen, thats the current default footer
### Horizontal mode Before/After Screenshots ### 
Before                     |  After
:-------------------------:|:-------------------------:
![before](https://user-images.githubusercontent.com/52841446/68287868-7031b400-00a9-11ea-814f-a94f043866dd.png) | ![after](https://user-images.githubusercontent.com/52841446/68287869-7031b400-00a9-11ea-8115-9b1dfc492bf7.png)